app/app.g.less

```diff
@@ -327,14 +327,14 @@ select {
 }
 
 ::-webkit-scrollbar-track {
-  background: @day-border;
+  background: var(--section);
   border-radius: 0 3px 3px 0;
 }
 
 ::-webkit-scrollbar-thumb {
-  background-color: @white;
-  border: 2px solid @day-border;
-  border-right: 1px solid @day-border;
+  background-color: var(--scrollbar);
+  border: 2px solid var(--border);
```

Comment:
I think this one should be called `border-style`

Reply:
can you explain your thoughts here? why border and not suffixing any of the other colors?